composables/use-fetch-with-csrf.ts

```diff
@@ -141,7 +146,10 @@ export const useFetchWithCsrf = createSharedComposable(() => {
 
       options.headers = headers;
       options.baseURL = baseUrl;
-      options.body = convertKeys(options.body, false, false);
+      // Only convert keys for non-FormData bodies
+      if (!isFormData && options.body !== undefined && options.body !== null) {
```

Comment:
before we didn't have the undefined and null checks is there an instance where the body is null or undefined and not FormData?